### PR TITLE
Update FAQ for usage-based pricing

### PR DESCRIPTION
**Pull request by mdubus**

### CODE REVIEW

File: data/pricing.js [hunk 1]
```diff
@@ -297,10 +319,30 @@ const pricingPageData = {
...
+      {
+        question: 'How do you calculate the number of documents?',
+        answer:
+          'The number of documents calculated for billing purposes is the total number of documents across all indexes in a project at the time of billing.',
+      },
       {
         question: 'Can I change my plan?',
         answer:
-          'Yes, and it is effortless. You can upgrade a project’s plan by selecting the project, navigating to the "Billing" tab, and clicking on the "Change plan" button. Please remember that once the process is complete, you won’t be able to downgrade your plan.',
+          'Yes, and it is effortless. You can upgrade a project’s plan by selecting the project, navigating to the “Billing” tab, and clicking on the “Change plan” button. Please remember that once the process is complete, you won’t be able to downgrade your plan.',
```
Comment (davelarkan) [inline data/pricing.js:335]:
```suggestion
          'Yes, and it is effortless. You can change a project’s plan by selecting the project, navigating to the “Billing” tab, and clicking on the “Change plan” button. Please remember that once the process is complete, you won’t be able to downgrade your plan.',
```

**Review by davelarkan — CHANGES_REQUESTED** [review]
Just a few minor tweaks (I'll update them in Notion too)

Other than that LGTM

**Review by bidoubiwa — APPROVED** [review]
🔥

**Comment by mdubus** [comment]
bors merge

**Pull request merged**